bin/clover/src/cli.ts

```diff
@@ -46,11 +52,50 @@ To generate all specs containing "ECS" or "S3", you can pass some services as ar
     .env(
       "SI_MODULE_INDEX_URL=<value:string>",
       "Set the module index url; defaults to http://0.0.0.0:5157",
-      { prefix: "SI_" }
+      { prefix: "SI_" },
+    )
+    .action(async (options, ...services: string[]) => {
+      await generateSiSpecs(
+        {
+          ...options,
+          moduleIndexUrl: options.moduleIndexUrl ?? DEFAULT_MODULE_INDEX_URL,
+          services: services?.length > 0 ? services : undefined,
+        },
+      );
+    })
+    // infer-ai
+    .command(
+      "infer-ai [...services:string]",
+      `Prompt the AI to give existing Cloud Formation specs based on their descriptions.
+
+To run inference on all specs:
+
+  clover infer-ai
+
+To run inference on all specs whose type names contain "ECS" or "S3", you can pass some services as arguments:
+
+  clover infer-ai ECS S3
+
+By default, the AI will only run inference for things that have not already been inferred. To
+force the AI to re-infer all specs, pass the --force flag.
+`,
+    )
+    .option("--force", "Force re-inference on cached values")
+    .option("--inferred <file:string>", "Inferred database location", {
+      default: "inferred.json",
+    })
```

Comment:
Is this a duplicate field or am I misreading this?

Reply:
Just an option you can pass to multiple subcommands (but not all of them). Cliffy is kind of unintuitive: each time you see `.command()` it starts another block of options. I looked to see what the right way was, but I think we may be using a different version from the modern docs.